CHANGELOG.md

```diff
@@ -89,6 +89,7 @@
 * [ENHANCEMENT] Added metrics `cortex_config_hash` and `cortex_runtime_config_hash` to expose hash of the currently active config file. #2874
 * [ENHANCEMENT] Logger: added JSON logging support, configured via the `-log.format=json` CLI flag or its respective YAML config option. #2386
 * [ENHANCEMENT] Clarify limitations of the `/api/v1/series`, `/api/v1/labels` and `/api/v1/label/{name}/values` endpoints. #2953
+* [ENHANCEMENT] Experimental Delete Series: Retry processing of Delete requests during failures. #2926
```

Comment:
We're cutting Cortex `1.3.0` and this PR will be merged after that. Please rebase `master` and move the CHANGELOG entry under the `master / unreleased` section at the top of the CHANGELOG. Thanks!